doc/Building_FRR_on_Ubuntu1604.md

```diff
@@ -0,0 +1,111 @@
+Building FRR on Ubuntu 12.04LTS from Git Source
+===============================================
+
+- MPLS is not supported on `Ubuntu 12.04` with default kernel. MPLS requires 
```

Comment:
I saw it here:
https://wiki.ubuntu.com/Kernel/Support#A16.04.x_Ubuntu_Kernel_Support
but actually it doesn't say "default", so if you checked that it's 4.4 forget my comment (I'm on a Debian now, but I'll check at home where I have an Ubuntu). Anyway, the note about kernel 4.5+ required doesn't hurt so let's leave it.